specification/compute/resource-manager/Microsoft.Compute/DiagnosticRP/preview/2024-03-01-preview/examples/GetDiagnosticOperation.json

```diff
@@ -0,0 +1,30 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "location": "eastus",
+    "operationId": "{operation-id}",
+    "api-version": "2024-03-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "message": "DiskInspection for selected resource succeeded.",
+        "responseFields": null,
```

Comment:
>null [](http://example.com/codeflow?start=26&length=4)

just wondering, its usually discouraged to include explicit nulls in the response, as opposed to omitting them - do you need them in the example? (Are the fields x-nullable true?)

Reply:
Removed responseFields: null from example file - this field is not being utilized for Disk Inspection diagnostic operation. Will also update our API to omit responseFields in response when value is null